src/content/docs/new-relic-one/use-new-relic-one/ui-data/automaps.mdx

```diff
@@ -0,0 +1,195 @@
+</Callout>
+
+## Open automap [#open-automap]
+
+Automap appears on any **Related entities** pane for supported entity types:
+
+  1. In **Explorer**, insert a filter to find your entity/service (for example, `alerts`).
+  2. Click on the entity/service.
+  3. On the **Summary page**, go to **Related entities** in the right pane.
+  4. Click **Map view**: 
+  
+    ![Screenshot showing the Map view button](./images/map-view-button.png "Screenshot showing the Map view button")
+
+The **Map view** button is only available for the following types of entities:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>Entity type
+        
+      </th>
+      <th>Description</th>
+      <th>Icon</th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        APM|APPLICATION
+      </td>
+      <td>
+        Application service instrumented with a New Relic APM agent
+      </td>
+      <td>
+        ![Screenshot of application icon](./images/application-icon.png "Screenshot of application icon")
+      </td>
+      <td>
+        
+      </td>
+    </tr>
+    <tr>
+      <td>
+        BROWSER|APPLICATION
+      </td>
+      <td>Browser application instrumented with New Relic Browser agent</td>
+      <td>
+        ![Screenshot of browser application icon](./images/browser-application-icon.png "Screenshot of browser application icon")
+      </td>
+    </tr>
+    <tr>
+      <td>
+        EXT|SERVICE
+      </td>
+      <td>
+        Services instrumented with OpenTelemetry, including Pixie-instrumented services.
+      </td>
+      <td>
+        ![Screenshot of application icon](./images/application-icon.png "Screenshot of application icon")
+      </td>
+    </tr>
+    <tr>
+      <td>
+        MOBILE|APPLICATION
+      </td>
+      <td>
+        Mobile app instrumented with a New Relic agent
+      </td>
+      <td>
+        ![Screenshot of browser application icon](./images/browser-application-icon.png "Screenshot of mobile application icon")
+      </td>
+    </tr>
+    <tr>
+      <td>
+        NR1|WORKLOAD
+      </td>
+      <td>
+        [Workload](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster/)
+      </td>
+      <td>
+        ![Screenshot of workload icon](./images/workload-icon.png "Screenshot of workload icon")
+      </td>
+    </tr>
+    <tr>
+      <td>
+        SYNTH|MONITOR
+      </td>
+      <td>
+        Synthetic monitor
+      </td>
+      <td>
+        ![Screenshot of synthetic icon](./images/synthetic-icon.png "Screenshot of synthetic icon")
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+## Explore the history
+
+Unlike simple navigation maps, such as New Relic maps in context, automap helps you understand how problems developed by allowing you to scroll backwards and forwards in time. You can also screen out entities you're not interested in.
```

Comment:
should there be a link on "maps in context" to explain what those are?

Reply:
@stumblefiend, I added a link to maps in context.